### PR TITLE
Added CTemplar, #198

### LINKED ISSUE #198: [NEW] New alternative
[//]: # ( Fill out to the best of your ability.  )
[//]: # ( If an item is not applicable, such as HTTPS for an App, feel free to leave it blank. )
[//]: # ( Mark off checkbox items by putting an x in between the [ ] without spaces, i.e. `[x]` )

### Checklist

- [ ] **I am affiliated with this alternative**
- [x] **I have read the rules outlined in [CONTRIBUTING.md](https://github.com/tycrek/degoogle/blob/master/CONTRIBUTING.md)**
- [ ] **Include my name in [CONTRIBUTORS.md](https://github.com/tycrek/degoogle/blob/master/CONTRIBUTORS.md)**

### Details

- **General**
  - **Alternative name**: CTemplar
  - **Alternative URL**: https://ctemplar.com/
  - **5/9/14 eyes**: no
  - **Description**: Open-source and privacy oriented email provider
- **FLOSS**
  - Repository: https://github.com/ctemplar/
  - Active: yes
  - Download: F-Droid
  - Price: freemium
  - Self-host: no
- **Encryption**
  - HTTPS: yes
  - E2EE: yes
  - Modern Encryption: yes
- **Transparency**
  - Public ownership: Templar Software Systems Ltd., a Seychelles Limited Liability Company.
  - 3rd-party audits: not that I can find, but [claims](https://ctemplar.com/features/) to serve 100% audited and open-sourced code
- **Privacy**
  - Anonymous payments: yes
  - Analytics: no
  - Captcha: no
- **Extras**
  - .onion address: ctemplarpizuduxk3fkwrieizstx33kg5chlvrh37nz73pv5smsvl6ad.onion
  - Warrant canary: no

### PR DESCRIPTION
**Pull request by kabo**
Resolves #198

### CODE REVIEW

**Comment by kabo** [comment]
> Could you please remove all edits to README.md? Per [CONTRIBUTING.md](https://github.com/tycrek/degoogle/blob/master/CONTRIBUTING.md):
> 
> > As of May 25 2020, the `README.md` file is automatically generated. Please edit anything in `yaml/` or `md/`.

Hmmm... I didn't edit anything in README.md. The weird thing is when I try to push it says I need to pull because there are changes. So I run `git pull --rebase` as per usual, and then `git push`. I've no idea where those changes came from.

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 ### *I don't have anything to hide, but I don't have anything to show you either.*
 
-![Updated](https://img.shields.io/static/v1?label=updated&message=August%204th%202020&color=388E3C&style=flat-square)
+![Updated](https://img.shields.io/static/v1?label=updated&message=August%2030th%202020&color=388E3C&style=flat-square)
```
Comment (tycrek) [inline README.md:23]: Revert these changes

File: README.md [hunk 2]
```diff
@@ -335,6 +335,7 @@ When contributing, **please try to follow the rules outlined in [CONTRIBUTING.md
 | [Anonymize.com Email](https://anonymize.com/) |  | Paid secure email service provided by Epik.com. |
 | [Thexyz](https://www.thexyz.com/) | **5-eyes** | Thexyz is a Canadian company offering privacy-conscious business email. Premium webmail costs USD $2.95 per month. |
 | [Mailfence](https://www.mailfence.com) | **14-eyes** | A secure and private email provider much like Protonmail and Tutanota. There are no google analytics nor hidden cookies. There are options to pay anonymously via cryptocurrency. It features a wide array of collaboration tools, such as groups, calendar, contacts, documents with editing space. Offers free, paid, and freemium plans. Can pay with cryptocurrency and has a [warrant canary]https://blog.mailfence.com/transparency-report-and-warrant-canary/). Thanks @resto1231 |
+| [CTemplar](https://ctemplar.com/) |  | Privacy-focused email provider located in Iceland. |
```
Comment (tycrek) [inline README.md:338]: This change as well

**Review by tycrek — CHANGES_REQUESTED** [review]
Sorry for the (very large) delay. It seems there are still changes in README.md on your branch. I would attempt to fix them myself but the GitHub web editor is being a PITA today.

**Comment by kabo** [comment]
The @actions-user didn't mess stuff up this time by the looks of it. Better now @tycrek ?

**Comment by tycrek** [comment]
I think I finally know where that issue is coming from on the last line in the README. Testing something to confirm.

**Comment by tycrek** [comment]
Didn't fix it. I'll merge this and hope nothing breaks.

**Pull request merged**